VultisigApp/VultisigApp/Chains/thorchain.swift

```diff
@@ -47,12 +81,11 @@ enum THORChainHelper {
         input.accountNumber = accountNumber
         input.sequence = sequence
         input.mode = .sync
-        // THORChain fee is 0.02 RUNE
         input.fee = CosmosFee.with {
-            $0.gas = 20000000
+            $0.gas = THORChainHelper.THORChainGas
             $0.amounts = [CosmosAmount.with {
                 $0.denom = "rune"
-                $0.amount = "2000000"
+                $0.amount = THORChainHelper.THORChainGas.description
```

Comment:
This amount should be put into `KeysignPayload.BlockSpecific.THORChain` , otherwise , in some case , main device and pair device will not agree with each other , this will cause tx can't be signed

Reply:
Moved to ```KeysignPayload.BlockSpecific.THORChain```